meta-xt-prod-devel-rcar-driver-domain/recipes-kernel/linux/files/defconfig

```diff
@@ -635,6 +635,7 @@ CONFIG_ACPI_TABLE_UPGRADE=y
 # CONFIG_ACPI_DEBUG is not set
 # CONFIG_ACPI_PCI_SLOT is not set
 CONFIG_ACPI_CONTAINER=y
+# CONFIG_ACPI_HOTPLUG_MEMORY is not set
```

Comment:
The commit message really does not explain what is happening

Reply:
In order to use Xen mapping optimizations (extended regions, etc) we need to enable several configs this is what current patch is doing. 
The main configs to add are:
+CONFIG_XEN_UNPOPULATED_ALLOC=y
+CONFIG_MEMORY_HOTPLUG=y
+CONFIG_MEMORY_HOTREMOVE=y
+CONFIG_ZONE_DEVICE=y
The main config to remove (as absolete) is:
-CONFIG_XT_CMA_HELPER=y
Other changes are done automatically.

I will mention that in the commit description.

Reply:
"+"

Reply:
"+"